pytripgui/app_logic/app_callbacks.py

```diff
@@ -63,9 +66,20 @@ def on_open_voxelplan(self) -> None:
         Returns:
         None
         """
+
+        def inline_thread_method():
+            app_controller.open_voxelplan(path)
+            progress_dialog.set_info_label_text('udalo sie')
+            progress_dialog._ui.ok_button.setEnabled(True)
```

Comment:
_ui jest prywatne, nie powinien być wykorzystywany poza klasą, która tworzy ten obiekt.

Reply:
tak, wiemy, jest to wersja pre-pre-alfa, która nie działa lub działa źle, a poza tym dobieranie się do prywatnych pól nie jest w tym projekcie niczym dziwnym :)

Reply:
nie zauważyłem "draft", sorki :dagger: :D

Reply:
![jojo-anime](https://user-images.githubusercontent.com/44035398/145999345-b743e430-1d76-44b4-85f1-132bb2645e60.gif)